pkg/selinux/selinux.go

```diff
@@ -409,19 +409,21 @@ func GetLxcContexts() (processLabel string, fileLabel string) {
 	}
 
 	if processLabel == "" || fileLabel == "" {
-		return "", ""
+		return "", "", nil
```

Comment:
Isn't this one (and the two above) an error condition? Why are they returning an nil error?